### ISSUE TITLE
Code shelter?

### ISSUE THREAD

**Issue by AndreaCrotti** [ptaoussanis/timbre#279]
Since this project seems a bit abandoned (forgive me if it's not the case), maybe it's worth to add it to the code shelter.

https://www.codeshelter.co/projects/add/
Which would allow some other maintainers to step in and help out with issues and PRs.
What you do think @ptaoussanis ?

**Comment by ptaoussanis** [comment]
Hi @AndreaCrotti, what leads you to believe the project is abandoned? The last commit was 4 months ago.

I'm pretty busy at the moment, so I tend to try batch my open source work- and there's nothing particularly urgent outstanding on Timbre right now?

**Comment by AndreaCrotti** [comment]
Ok sorry for the assumption @ptaoussanis.
Maybe nothing urgent but the problems I found are that I can't actually have test coverage in a project which uses Timbre, more info in this issue https://github.com/ptaoussanis/timbre/issues/274

So it kind of seems abandoned if even the last version in master is using `cljx` files which have been deprecated for a long time, which also seems to be the source of the problem with cloverage.
(even if I can't reproduce the same issue on another project for some reason).

Then I noticed that there was no CI set up and I was just trying to set up tests locally, and I filed this other issue when I could not get that working https://github.com/ptaoussanis/timbre/issues/277
